packages/features/ee/workflows/api/scheduleSMSReminders.ts

```diff
@@ -99,6 +101,37 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           booking: reminder.booking,
         });
 
+        // checks whether booking belongs to a organization and updates rescheduleLink.
+        let rescheduleLink = WEBAPP_URL;
```

Comment:
I think we can do this a lot easier and use the same logic as in handleNewBooking.ts to get the bookerUrl, see here: https://github.com/calcom/cal.com/blob/main/packages/features/bookings/lib/handleNewBooking.ts#L1452